templates/alertmanager.service.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Prometheus Alertmanager
+After=network.target
+
+[Service]
+WorkingDirectory={{ prometheus_alertmanager_db_path }}
```

Comment:
Why do not you use `-storage.path={{ prometheus_alertmanager_db_path }}` instead?